### ISSUE TITLE
in FlareConnect the volume knob of a Slave lost function until device muted/unmuted

### ISSUE THREAD

**Issue by lin0lin0** [mkulesh/onpc#293]
As I just updated/installed MusicControl v. 1.31 I was notified for Firmware update of my AVR. and I did it (never change a running system).
Before I always used the Pioneer RemoteApp v 2.0.
So I do not know if I did something wrong or the Update of the AVR or the "Music Control" changed something.

Before I was able to use volume knob of my three "Slaves" devices. since using Music Control and/or Update of AVR each: one NCP-302 and one MRX-5 the HW Vol. knob is not working unless I set the Slave device in App to mute and back. For one other NCP-302 the Vol. knob still is working if Slave or standalone. If once made active the Vol. knob via mute workaround and putting Slave to sleep and back on the Vol. knob is inactive again until Slave was muted...

Did I just do something wrong, or anybody knows a solution

**Comment by mkulesh** [comment]
Hi @lin0lin0,
please visit https://github.com/mkulesh/onpc/releases/tag/v1.28, download and manually install onpc-v1.28-release.apk
While the version will be downgraded, you may need to uninstall the existing latest app version.
Do you observe the same problem in the version 1.28? If yes, the problem is with the AVR firmware and we are not able to fix it. If the problem disappears in the v1.28, I will try to analyze it and will ask you later about app logs.

**Comment by lin0lin0** [comment]
Hi Mikhel, 
thanks for suggestion. But old version 1.28 did not change behavior regarding the Vol. knob. Also the Official Pioneer "Remote App" v2.0 behaves the same.

But back on 1.31 I did a (same release) FW update on my two "issue" Slaves again. Was strange to me that also Master stopped Deezer playback (not in FlareConnect) as I started the FW update to a Slave.
After 1st cold-restart+Flare both updated Slaves did react on Vol. knob, but after sleep and wake-up only one keeps unresponsive to the Vol. knob.
So I assume it is the Pioneer/Onkyo FW which is the culprit part.
Later I will try again FW update on the remaining Slave - hence it is now just one out of three Slaves I'm confident it could be changed.
br
